src/vs/workbench/contrib/preferences/browser/keybindingsEditor.ts

```diff
@@ -135,10 +136,16 @@ export class KeybindingsEditor extends BaseEditor implements IKeybindingsEditor
 		this.createBody(keybindingsEditorElement);
 	}
 
-	setInput(input: KeybindingsEditorInput, options: EditorOptions | undefined, token: CancellationToken): Promise<void> {
+	setInput(input: KeybindingsEditorInput, options: SettingsEditorOptions | undefined, token: CancellationToken): Promise<void> {
```

Comment:
`SettingsEditorOptions` is reused here for that I found [`PreferencesEditor`](https://github.com/microsoft/vscode/blob/00ee3285eb4153ae2cc400785573ede9bd6ec5bd/src/vs/workbench/contrib/preferences/browser/preferencesEditor.ts#L155-L163) also uses it in `setInput`, and also only uses `query` among the extended properties.

Reply:
I would suggest to reuse `IKeybindingsEditorSearchOptions` instead of using `SettingsEditorOptions`. 

https://github.com/microsoft/vscode/blob/6eaf748366a48a5b6f9645cfa5e2257f51a027a6/src/vs/workbench/services/preferences/common/preferencesEditorInput.ts#L52-L55